buildpipeline/pipeline.json

```diff
@@ -16,69 +16,7 @@
       },
       "Definitions": [
         {
-          "Name": "Core-Setup-Linux-BT",
-          "Parameters": {
-            "PB_DistroRid": "debian.8-x64",
-            "PB_DockerTag": "debian-8.2-debpkg-9f87c3c-20173003023006",
-            "REPO_ID": "579f8fb0fedca9aeeb399132",
-            "REPO_USER": "dotnet",
-            "REPO_SERVER": "azure-apt-cat.cloudapp.net"
-          },
-          "ReportingParameters": {
-            "OperatingSystem": "Debian 8.2",
-            "Type": "build/product/",
-            "Platform": "x64"
-          }
-        },
-        {
-          "Name": "Core-Setup-Linux-BT",
-          "Parameters": {
-            "PB_DistroRid": "ubuntu.14.04-x64",
-            "PB_DockerTag": "ubuntu-14.04-debpkg-e5cf912-20175003025046",
-            "REPO_ID": "562fbfe0b2d7d0e0a43780c4",
-            "REPO_USER": "dotnet",
-            "REPO_SERVER": "azure-apt-cat.cloudapp.net",
-            "CLI_NUGET_FEED_URL": "https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v2/package",
-            "CLI_NUGET_SYMBOLS_FEED_URL": "https:%2F%2Fdotnet.myget.org/F/dotnet-core/symbols/api/v2/package"
-          },
-          "ReportingParameters": {
-            "OperatingSystem": "Ubuntu 14.04",
-            "Type": "build/product/",
-            "Platform": "x64"
-          }
-        },
-        {
-          "Name": "Core-Setup-Linux-BT",
-          "Parameters": {
-            "PB_DistroRid": "ubuntu.16.04-x64",
-            "PB_DockerTag": "ubuntu-16.04-debpkg-e5cf912-20174703024721",
-            "REPO_ID": "575f40f3797ef7280505232f",
-            "REPO_USER": "dotnet",
-            "REPO_SERVER": "azure-apt-cat.cloudapp.net"
-          },
-          "ReportingParameters": {
-            "OperatingSystem": "Ubuntu 16.04",
-            "Type": "build/product/",
-            "Platform": "x64"
-          }
-        },
-        {
-          "Name": "Core-Setup-Linux-BT",
-          "Parameters": {
-            "PB_DistroRid": "ubuntu.16.10-x64",
-            "PB_DockerTag": "ubuntu-16.10-debpkg-ec863bb-20170003030028",
-            "REPO_ID": "575f40f3797ef7280505232f",
-            "REPO_USER": "dotnet",
-            "REPO_SERVER": "azure-apt-cat.cloudapp.net"
-          },
-          "ReportingParameters": {
-            "OperatingSystem": "Ubuntu 16.10",
-            "Type": "build/product/",
-            "Platform": "x64"
-          }
-        },
-        {
-          "Name": "Core-Setup-Linux-BT",
```

Comment:
Missed it, will take care of it. Thanks. Will have Core-Setup-Linux-Arm-BT and Core-Setup-Linux-BT